int16 state_fsm
int16 mode_ai

int16 positionx
int16 positiony
int16 positiont

int16 ballposx
int16 ballposy
int8 balldetect
uint8 state

int16 lostPackets

int16 value